docs/query-layer/quality.md

```
@@ -0,0 +1,69 @@
+id: 019f53ab-e7a9-7c71-b13d-eddf64cc07fe
+slug: specs/query-layer/quality
+title: "Query layer — quality bar and evidence"
+type: spec
+status: active
+priority: high
+tags: [graphql, query-layer, spec]
+---
+
+# Quality bar
+
+### Test plan (files, per phase)
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)